packages/material-ui-icons/builder.js

```diff
@@ -160,7 +159,7 @@ async function worker({ svgPath, options, renameFilter, template }) {
 
   if (!exists2) {
     console.log(`Making dir: ${outputFileDir}`);
-    mkdirp.sync(outputFileDir);
+    fse.mkdirpSync(outputFileDir);
```

Comment:
```suggestion
    await fse.mkdirp(outputFileDir);
```

Could technically replace this entire block with just that one line
https://github.com/mui-org/material-ui/blob/c6116c807df810ed32154422890d3e6d94fa406d/packages/material-ui-icons/builder.js#L157-L163

Reply:
Keeping it equivalent to be safe.